packages/math-new/README.md

```diff
@@ -0,0 +1,11 @@
+# Overview
+The Balancer SOR Economics Extension is a package designed to extend the functionality of the Balancer SOR package. This extension introduces essential economics concepts, such as price impact and effective price calculations, into the package.
+
+# Package Structure
+The package is structured around a main Automated Market Maker (AMM) class that serves as the central point of interaction for users. The main components of the package are:
+
+- AMM Class: The AMM class incorporates all the economic functions related to the Balancer SOR pools. It provides methods to calculate price impact, effective price, and other economic indicators for trades.
+
+- SOR Extended Pool Classes: The package uses the Balancer SOR Pool classes as attributes in the AMM class. These classes from the Balancer SOR pools package handle the core functionality of interacting with Balancer SOR pools. Usually, we need to add a few functions on the SOR package (as an spot price function). Those functions should use Balancer SOR code as reference or even copied.
```

Comment:
Make sure we're referencing them back when copied and to mention that we've done that because they're not exported in the main package. I'd also consider forking the SOR repo ourselves and introducing some of these changes there -- at least the ones regarding exporting of functions (like @luizakp did [here](https://github.com/bleu-studio/balancer-sor/pull/1)) -- and making a PR back to their main repo, so that we don't have to worry about changes to their code not being reflected back on our side. WDYT?

Would also take a look into https://github.com/balancer/b-sdk which seems to be the unification of the SOR with the SDK and peek into what they're set out to do there -- I'd be curious about whether could help that effort somehow.